nilearn/regions/signal_extraction.py

```diff
@@ -39,19 +194,20 @@ def img_to_signals_labels(imgs, labels_img, mask_img=None,
 
     mask_img : Niimg-like object, optional
         See :ref:`extracting_data`.
-        Mask to apply to labels before extracting signals. Every point
-        outside the mask is considered as background (i.e. no region).
+        Mask to apply to labels before extracting signals.
+        Every point outside the mask is considered
+        as background (i.e. no region).
 
     background_label : number, optional
-        Number representing background in labels_img. Default=0.
+        Number representing background in labels_img.
```

Comment:
I'm not in favor of removing default values.

Reply:
default values are back